network/src/counters.rs

```diff
@@ -87,6 +87,15 @@ pub static LIBRA_NETWORK_DIRECT_SEND_BYTES: Lazy<HistogramVec> = Lazy::new(|| {
     .unwrap()
 });
 
+pub static LIBRA_NETWORK_NOISE_HANDSHAKE: Lazy<IntCounterVec> = Lazy::new(|| {
+    register_int_counter_vec!(
+        "libra_network_noise_handshake",
+        "Libra network noise handshake counter",
+        &["role_type", "network_type"]
```

Comment:
i think we're already using "role_type" and "network_type" for different purposes in other counters?

Reply:
I can't re-use the same labels across counters?

Reply:
err no, i think it's just confusing to use `role_type := initiator | responder` in one place and `role_type := validator | fullnode` in another. what i'm suggesting is to rename "role_type" and "network_type" for this specific counter since labels with the same name are being used differently in other counters.